ToolBox/IQFeed/IQFeedDataQueueHandler.cs

```diff
@@ -332,6 +329,13 @@ public bool CanAdvanceTime(SecurityType securityType)
         {
             return _symbolUniverse.CanAdvanceTime(securityType);
         }
+
+        /// <summary>
+        /// Performs application-defined tasks associated with freeing, releasing, or resetting unmanaged resources.
+        /// </summary>
+        public void Dispose()
```

Comment:
IQFeedDQH seems to be missing a lot of Dispose calls and some refactoring to achieve it too, maybe we can generate a new issue just for this?

Reply:
Sure, we can create a new issue when this one is merged 👍